app/src/main/java/com/pajato/android/gamechat/exp/model/Checkers.java

```diff
@@ -0,0 +1,185 @@
+package com.pajato.android.gamechat.exp.model;
+
+import com.google.firebase.database.Exclude;
+import com.google.firebase.database.IgnoreExtraProperties;
+import com.pajato.android.gamechat.exp.ExpType;
+import com.pajato.android.gamechat.exp.Experience;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.pajato.android.gamechat.exp.ExpType.checkers;
+
+/** Provide a Firebase model class for a checkers game experience. */
+@IgnoreExtraProperties
+public class Checkers implements Experience {
+
+    public final static int ACTIVE = 0;
+    public final static int PRIMARY_WINS = 1;
+    public final static int SECONDARY_WINS = 2;
+    public final static int TIE = 3;
+    public final static int PENDING = 4;
+
+    /**
+     * A map of board position (0->63) to piece type (where piece type can be PRIMARY_PIECE,
+     * PRIMARY_KING, SECONDARY_PIECE or SECONDARY_KING).
+     */
+    public Map<String, String> board;
```

Comment:
I have a hunch our notion of a "board" will expand to include colors (red/black for checkers, white/black for chess), shapes, sizes, etc.  I see it emerging as a simple pojo containing lists of player pieces.  As a model this works for Go, TTT, Chess and Checkers, probably more.   Just thinking out loud, so to speak.